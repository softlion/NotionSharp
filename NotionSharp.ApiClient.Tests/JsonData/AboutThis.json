{
  "results": [
    {
      "id": "c7b44455-3d31-4a5b-b82c-b7e3d85ba585",
      "created_time": "2020-04-30T13:37:00+00:00",
      "last_edited_time": "2023-11-26T16:07:00+00:00",
      "archived": false,
      "created_by": {
        "id": "ab9257e1-d027-4494-8792-71d90b63dd35",
        "object": "user"
      },
      "last_edited_by": {
        "id": "ab9257e1-d027-4494-8792-71d90b63dd35",
        "object": "user"
      },
      "url": "https://www.notion.so/About-this-c7b444553d314a5bb82cb7e3d85ba585",
      "public_url": "https://wise-spirit-737.notion.site/About-this-c7b444553d314a5bb82cb7e3d85ba585",
      "parent": {
        "type": "page_id"
      },
      "properties": {
        "title": {
          "title": [
            {
              "type": "text",
              "plain_text": "About this",
              "annotations": {
                "bold": false,
                "italic": false,
                "strikethrough": false,
                "underline": false,
                "code": false,
                "color": "default"
              },
              "text": {
                "content": "About this"
              }
            }
          ],
          "id": "title",
          "type": "title"
        }
      },
      "object": "page"
    }
  ],
  "has_more": false,
  "object": "list"
}