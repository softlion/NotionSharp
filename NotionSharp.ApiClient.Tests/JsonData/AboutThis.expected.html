<div class="notion-paragraph"><div class="notion-line">This content is written on a Notion page. It is a demo page for our csharp </div><div class="notion-line"><a href="http://notion.so/"><a href="http://notion.so/">notion.so</a></a></div><div class="notion-line"> api client.</div></div>
<div class="notion-paragraph"></div>
<h1><div class="notion-line">Some API</div></h1>
<div class="notion-paragraph"><div class="notion-line">It contains some lines to test the api client, especially these methods:</div></div>
<ul><li><div class="notion-line">GetHtml</div></li></ul>
<ul><li><div class="notion-line">LoadPageChunk</div></li></ul>
<ul><li><div class="notion-line">GetSyndicationFeed</div></li></ul>
<div class="notion-paragraph"></div>
<div class="notion-paragraph"><div class="notion-line">â™“ The last one is the most powerful: it creates a rss like feed from a </div><div class="notion-line"><a href="http://notion.so/"><a href="http://notion.so/">notion</a></a></div><div class="notion-line"> collection. It opens cool things like creating a full website from a notion page hierarchy.</div></div>
<div class="notion-paragraph"></div>
<div class="notion-callout-block notion-block-color-graybackground"><img class="notion-emoji" src="https://cdn.jsdelivr.net/gh/twitter/twemoji/assets/svg/1f4e2.svg" />
<div class="notion-line">This is a CallOut !</div></div>
<div class="notion-paragraph"></div>
<div class="notion-column_list-block"><div style="display: flex"><div class="notion-column" style="width: calc((100% - 46px) * 1);"><div class="notion-paragraph"><div class="notion-line"><span class=" notion-color-Orange notion-code">First Column (25%)
</span></div></div>
</div><div class="notion-column-separator"><div class="notion-column-separator-line"></div></div><div class="notion-column" style="width: calc((100% - 46px) * 1);"><div class="notion-paragraph"><div class="notion-line"><span class=" notion-color-Green notion-code">Second column (75%)</span></div></div>
<div class="notion-image-block"><img src="https://prod-files-secure.s3.us-west-2.amazonaws.com/da89a8e0-9c55-4c13-9966-796e0f0c1bac/3b632032-656f-4e87-87b7-b95bcb786f9d/Untitled.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45HZZMZUHI%2F20231127%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20231127T072100Z&X-Amz-Expires=3600&X-Amz-Signature=ae13232280da11ebbf30f754f3e48f26f6023621f97b9f7b812fd86ac6bebef7&X-Amz-SignedHeaders=host&x-id=GetObject"/></div>
</div></div></div><div class="notion-paragraph"></div>
<div class="notion-quote-block notion-block-color-default>"><div class="notion-line">The End Quote</div></div>
<div class="notion-paragraph"></div>
